cpp/src/svm/svc_impl.cuh

```diff
@@ -72,18 +72,16 @@ void svcFitX(const raft::handle_t& handle,
   {
     rmm::device_uvector<math_t> unique_labels(0, stream);
     model.n_classes = raft::label::getUniquelabels(unique_labels, labels, n_rows, stream);
-    rmm::device_async_resource_ref rmm_alloc = rmm::mr::get_current_device_resource();
-    model.unique_labels                      = (math_t*)rmm_alloc.allocate_async(
-      model.n_classes * sizeof(math_t), rmm::CUDA_ALLOCATION_ALIGNMENT, stream);
-    raft::copy(model.unique_labels, unique_labels.data(), model.n_classes, stream);
+    model.unique_labels->resize(model.n_classes * sizeof(math_t), stream);
+    raft::copy((math_t*)model.unique_labels->data(), unique_labels.data(), model.n_classes, stream);
```

Comment:
Reference: https://github.com/rapidsai/rmm/blob/c494395e58288cac16321ce90e9b15f3508ae89a/include/rmm/device_uvector.hpp#L414

```suggestion
    model.unique_labels = unique_labels.release()

Reply:
Ah, got it